bin/asciibinder

```diff
@@ -180,17 +221,41 @@ else
   elsif not repo_dir.writable?
     Trollop::die "The specified repo directory '#{repo_dir}' cannot be written to."
   else
-    ['.git','_build_cfg.yml','_distro_map.yml','_templates'].each do |file|
-      if not File.exist?(File.join(repo_dir, file))
-        Trollop::die "The specified repo directory '#{repo_dir}' does not appear to be an AsciiBinder repo."
-      end
-    end
+    repo_check(repo_dir)
   end
 end
 
 # Set the repo root
 set_source_dir(File.expand_path(repo_dir))
 
+# Cloning? Time to try it.
+if cmd == 'clone'
+  puts "Cloning #{cmd_opts[:giturl]} to #{repo_dir}"
+  stdout, stderr, procstat = Open3.capture3("git clone #{cmd_opts[:giturl]} #{repo_dir}")
```

Comment:
This can be done a lot simpler without open3...

```ruby
puts "Cloning #{cmd_opts[:giturl]} to #{repo_dir}"
system("git clone #{cmd_opts[:giturl]} #{repo_dir}")
Trollop::die "The git URL could not be cloned: #{err}" if $?.exitstatus != 0
```